code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm

```diff
@@ -386,7 +386,7 @@
 	return ..()
 
 /datum/action/ability/xeno_action/rage/can_use_action(atom/A, silent = FALSE, override_flags)
-	. = ..()
+	. = ..(silent, override_flags)
```

Comment:
This is not a real fix.
Args are meant to be static for a reason, you need to fix the args for this proc generally, not snowflake a fix in.